realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -1048,6 +1081,151 @@ void BaseRealSenseNode::publishStaticTransforms()
     }
 }
 
+/* Modified for Brain */
+static void output_unit_vector(unsigned int height, unsigned int width, float unitVectors[FRAME_MAX_HEIGHT][FRAME_MAX_WIDTH * 3])
+{
+	unsigned int i, j;
+	FILE *fp; /* output to .obj file with a certain format to be shown by some tool */
+
+	std::cerr << "\nUnit Test for Realsense Unit Vector:\nOutput the saved data: frame width is " \
```

Comment:
We should use [ROS's built in logging](http://wiki.ros.org/roscpp/Overview/Logging#rosconsole.2BAC8-CodeAPI.line-112) macros to be consistent with the rest of the node (especially since these aren't errors). For most of these _cerr_ streams, I imagine we could just use ROS_INFO or ROS_DEBUG.